codecov.yml

```diff
@@ -1,2 +1,10 @@
 codecov:
   branch: main # set new Default branch
```

Comment:
```suggestion
```

I think maybe we should delete this for now...it might cause inconsistency once copied to other branches, wdyt?

Reply:
why inconsistency? it was first defined on yesterday's PR, there are no other versions

Reply:
The docs for this field says:

> Codecov will use the default branch from Git for your repository as the primary source of the repositories' Codecov.yml. 

I'm just wary that when this gets copied to branches like `rhoai`, that runs against that branch (or PRs to that branch) should use the codecov.yml from that branch, not necessarily from main.

To counter-question: what is the value of specifying this here?

Reply:
Basically it is to have a single-point of configuration. 
If both the Global YAML AND the Repository YAML are set, the Repository-level YAML has priority.

Reply:
The default branch should be taken automatically from the repo based on the doc, so we may leave the file here to have a repository config yaml available but I would try removing the branch part, so the default branch is taken automatically from the repo.